README.md

```diff
@@ -14,6 +14,8 @@ Run `ng generate component component-name` to generate a new component. You can
 
 `ng build --configuration production --aot --output-hashing=none`
 
+`./compressBuild.sh` compresses the build
+
```

Comment:
This should maybe be under the other `--` arguments since they are part of the `ng build`